var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -30,12 +30,12 @@ class Ucx(AutotoolsPackage):
     MPI/PGAS frameworks"""
 
     homepage = "http://www.openucx.org"
-    url      = "https://github.com/openucx/ucx/releases/download/v1.2.1/ucx-1.2.1.tar.gz"
-
+    url      = "https://github.com/openucx/ucx/releases/download/v1.3.1/ucx-1.3.1.tar.gz"
```

Comment:
@floquet unless the URL format changes, you shouldn't have to update the URL when you add a version (it isn't harmful though)